Cargo.toml

```diff
@@ -31,7 +31,7 @@ members = [
 	"crates/nu-protocol",
 	"crates/nu-plugin",
 	"crates/nu_plugin_inc",
-	"crates/nu_plugin_gstat",
+	#"crates/nu_plugin_gstat",
```

Comment:
I'd prefer not to lose the gstat plugin.

Reply:
Sorry, that was a testing thing I had to do to get `cargo test --workspace` to run.